plugins/streams/redis/src/hopeit/redis_streams/__init__.py

```diff
@@ -151,33 +151,32 @@ async def read_stream(self, *,
         :return: yields Tuples of message id (bytes) and deserialized DataObject
         """
         try:
-            batch = await self._read_pool.xread_group(
-                consumer_group,
-                self.consumer_id,
-                streams=[stream_name],
-                latest_ids=[offset],
+            batch = await self._read_pool.xreadgroup(
```

Comment:
Variable naming suggestion: rename `batch` to `response`

then instead of `items`, use `batch` to keep same meaning as previous code.